### DISCUSSION TITLE
[FEATURE REQUEST] Lock comments

### DISCUSSION THREAD

**Discussion by atomicpoet** [mastodon/mastodon#21446] [General]
If comments are allowed, moderators and posters should have the option to lock comments thereby preventing more of them.

This is useful for when:

1. A thread gets overlong and continues indefinitely
2. A thread is dogpiled by harassers, trolls, and other malicious people
3. OP is unable to reply in a timely manner

**Comment by MattWilcox** [comment]
This seems useful, and also not open to producing skewed narratives (something that individual response moderation by a poster would allow, this does not)

**Comment by Cassolotl** [comment]
Related to *“Disable replies” feature #8565*.

**Comment by ghost** [comment]
Allowing moderators to lock comments on other people's posts is a recipe for administrative abuse. This should be an option for the posters themselves only.
